homeassistant/components/ozw/websocket_api.py

```diff
@@ -106,6 +113,63 @@ def _call_util_function(hass, connection, msg, send_result, function, *args):
     connection.send_result(msg[ID])
 
 
+@websocket_api.require_admin
+@websocket_api.async_response
+@websocket_api.websocket_command(
+    {
+        vol.Required(TYPE): "ozw/migrate_zwave",
+        vol.Optional(DRY_RUN, default=True): vol.Coerce(bool),
```

Comment:
No need to coerce, we're not relying on user data.
```suggestion
        vol.Optional(DRY_RUN, default=True): bool,
```